### PR TITLE
Add Imgur

### PR DESCRIPTION
**Pull request by betsythefc**

### CODE REVIEW

**Comment by RichJeanes** [comment]
How does Imgur meet our [criteria](https://github.com/2factorauth/twofactorauth/blob/master/CONTRIBUTING.md#site-criteria) for sites to be listed?

**Comment by betsythefc** [comment]
It seems I mis-read the criteria, sorry about that!

**Comment by Carlgo11** [comment]
Well it does have private messages so I'd say it qualifies. :+1:

**Comment by RichJeanes** [comment]
Imgur has PMs? Do people actually use it as a messaging platform?

**Comment by Carlgo11** [comment]
Yes people use PMs on Imgur. If not for normal messages then for nudes :disappointed: #WhyWouldYouEvenSendNudesOverTheInternet

File: _data/social.yml [hunk 1]
```diff
@@ -85,6 +85,12 @@ websites:
       twitter: instagram
       img: instagram.png
       tfa: No
+      
+    - name: Imgur
+      url: https://www.imgur.com
```
Comment (Carlgo11) [inline _data/social.yml]: The URL you entered redirects to `https://imgur.com/`. Please use that instead.

Comment (betsythefc) [inline _data/social.yml]: I made the change, thanks!

**Comment by Carlgo11** [comment]
Thank you!

**Comment by jamcat22** [comment]
Thanks for contributing @betsythefc!
While your changes have been merged with our repo, it can take some time before they show up on the website.
You can learn more about our merging process on our [wiki](https://github.com/2factorauth/twofactorauth/wiki/FAQ_Merge).

**Pull request merged**